ui/components/MeshConfigComponent.js

```diff
@@ -436,22 +435,21 @@ function MesherySettingsNew({ classes, updateProgress, operatorState, k8sconfig
         },
         customBodyRender: (_, tableMeta) => {
           return (
-            <Tooltip title={`Server: ${tableMeta.rowData[2]}`}>
-              <Chip
-                label={data[tableMeta.rowIndex].context}
-                onDelete={handleConfigDelete(
+            <ConnectionChip
+              tooltip={`Server: ${tableMeta.rowData[2]}`}
+              title={data[tableMeta.rowIndex].context}
+              onDelete={() =>
+                handleConfigDelete(
                   data[tableMeta.rowIndex].connection_id,
                   data[tableMeta.rowIndex].context,
                   tableMeta.rowIndex,
-                )}
-                onClick={() =>
-                  handleKubernetesClick(data[tableMeta.rowIndex].connection_id, tableMeta.rowIndex)
-                }
-                icon={<img src="/static/img/kubernetes.svg" className={classes.icon} />}
-                variant="outlined"
-                data-cy="chipContextName"
-              />
-            </Tooltip>
+                )
+              }
+              handlePing={() =>
+                handleKubernetesClick(data[tableMeta.rowIndex].connection_id, tableMeta.rowIndex)
```

Comment:
https://github.com/meshery/meshery/pull/9388/files#diff-9514a9ef762e5d87704d59bd30ae5c0a1744322ddcc1bda1a51034bacf4ff690R260 Delete this func, and use the k8s hook.